pkg/guacrest/openapi.yaml

```diff
@@ -72,34 +141,46 @@ paths:
           $ref: "#/components/responses/InternalServerError"
         "502":
           $ref: "#/components/responses/BadGateway"
-  "/analysis/dependencies":
+  "/v0/artifact/{digest}/vulns":
     get:
-      summary: Identify the most important dependencies
-      operationId: analyzeDependencies
+      summary: Get vulnerabilities for a specific digest
+      operationId: getArtifactVulns
       parameters:
-        - $ref: "#/components/parameters/PaginationSpec"
-        - name: sort
-          description: >
-            The sort order of the packages
-              * 'frequency' - The packages with the highest number of dependents
-              * 'scorecard' - The packages with the lowest OpenSSF scorecard score
-          in: query
+        - name: digest
+          in: path
           required: true
+          description: Digest, the second part from artifact identifier in the format <algorithm:digest>
           schema:
             type: string
-            enum:
-              - frequency
-              - scorecard
       responses:
         "200":
-          $ref: "#/components/responses/PackageNameList"
+          $ref: "#/components/responses/VulnerabilityList"
+        "400":
+          $ref: "#/components/responses/BadRequest"
+        "500":
+          $ref: "#/components/responses/InternalServerError"
+        "502":
+          $ref: "#/components/responses/BadGateway"
+  "/v0/artifact/{digest}/dependencies":
+    get:
+      summary: Get dependencies for a specific digest
+      operationId: getArtifactDeps
+      parameters:
+        - name: digest
+          in: path
+          required: true
+          description: Digest, the second part from artifact identifier in the format <algorithm:digest>
+          schema:
+            type: string
```

Comment:
Optional: we could also surface the `linkCondition` configuration for the dependency retrieval by digest.